### ISSUE TITLE
Connect to Ledger Nano X over bluetooth

### ISSUE THREAD

**Issue by ppppso** [m2049r/xmrwallet#597]
Ledger Nano X has bluetooth connectivity. Monerujo should be able to connect to the device over bluetooth.

**Comment by m2049r** [comment]
yes it should. i started looking into that. then stopped as i realized that in order to connect to bluetooth, the user has to give the app the permission for geo-location and didn't want to deal with the "monerujo is spying on me - wtf does it need to know where i am!"-shitstorm right now.

how would **you** feel about giving this permission to this app for no logic reason?

ledger live also asks for this permission, and states that they will not use the info for anything else ...

**Comment by SamsungGalaxyPlayer** [comment]
@m2049r it's obviously not ideal, but restricting the prompt for the access until someone attempts to connect to a Bluetooth device could be a good compromise for some people.

**Comment by ppppso** [comment]
> how would **you** feel about giving this permission to this app for no logic reason?

I would not mind if the permission was requested only when I want to enable the Bluetooth feature.

**Comment by m2049r** [comment]
I have the code almost done. The Ledger Monero App does not support Bluetooth yet - as soon as they are done, I can complete it.

**Comment by netnye** [comment]
Glad I searched github prior to posting. Also wondering about this. Would love to see this feature, and also think the go-location isn't a big deal if only done at time of Bluetooth connection (not upon install of app)

**Comment by ianmacd** [comment]
Here we are, 18 months later. It doesn't look as if it's ever going to happen, does it?

**Comment by zyrorl** [comment]
@m2049r i would really appreciate this feature man, it'd be really great. Other wallets support bluetooth with ledger, i'd love to see it with monerujo. Is there anything in the way of this feature being implemented other than privacy concerns?

**Comment by pricead** [comment]
As far as I know the hold up has nothing to do with privacy concerns. The Ledger device's XMR app (https://github.com/LedgerHQ/app-Monero) needs to be updated to add a Bluetooth interface, and only then can apps like Monerujo be updated to use BT.

**Comment by CryptoGrampy** [comment]
@m2049r do you know if there is a specific ticket for this feature filed with Ledger?  I tried searching through their Monero wallet issues and came up with nothing.  Would love to see this feature get added.... I can create a ticket with them, but I'm not really sure what to say.

**Comment by Roki100** [comment]
any updates?
